packages/multi-test/src/executor.rs

```diff
@@ -98,20 +98,20 @@ where
     /// Execute a contract and process all returned messages.
     /// This is just a helper around execute(),
     /// but we parse out the data field to that what is returned by the contract (not the protobuf wrapper)
-    fn execute_contract<T: Serialize>(
+    fn execute_contract<T: Serialize + std::fmt::Debug>(
         &mut self,
         sender: Addr,
         contract_addr: Addr,
         msg: &T,
         send_funds: &[Coin],
     ) -> AnyResult<AppResponse> {
-        let msg = to_binary(msg)?;
-        let msg = WasmMsg::Execute {
-            contract_addr: contract_addr.into(),
-            msg,
+        let binary_msg = to_binary(msg)?;
+        let wrapped_msg = WasmMsg::Execute {
+            contract_addr: contract_addr.to_string(),
```

Comment:
to_string is more explicit, but does require a clone.
`into_string`?

(Note: this is pedantic to even worry about performance here... this is test code after all... just my usual review bugs chipped in)

Reply:
Good catch.